docs/src/main/asciidoc/security-ldap.adoc

```diff
@@ -160,20 +160,22 @@ quarkus.security.ldap.enabled=true
 
 quarkus.security.ldap.dir-context.principal=uid=admin,ou=system
 quarkus.security.ldap.dir-context.url=ldaps://ldap.server.local <1>
+%test.quarkus.security.ldap.dir-context.url=ldap://127.0.0.1:10389 <2>
 quarkus.security.ldap.dir-context.password=secret
 
 quarkus.security.ldap.identity-mapping.rdn-identifier=uid
 quarkus.security.ldap.identity-mapping.search-base-dn=ou=Users,dc=quarkus,dc=io
 
 quarkus.security.ldap.identity-mapping.attribute-mappings."0".from=cn
-quarkus.security.ldap.identity-mapping.attribute-mappings."0".filter=(member=uid={0},ou=Users,dc=quarkus,dc=io) <2>
+quarkus.security.ldap.identity-mapping.attribute-mappings."0".filter=(member=uid={0},ou=Users,dc=quarkus,dc=io) <3>
 quarkus.security.ldap.identity-mapping.attribute-mappings."0".filter-base-dn=ou=Roles,dc=quarkus,dc=io
 
-%test.quarkus.security.ldap.dir-context.url=ldap://127.0.0.1:10389 <3>
+quarkus.security.ldap.cache.enabled=true <4>
 ----
 <1> You need to provide the URL to an LDAP server. This example requires the LDAP server to have imported link:{quarkus-blob-url}/test-framework/ldap/src/main/resources/quarkus-io.ldif[this LDIF file].
-<2> `{0}` is substituted by the `uid`.
-<3> The URL used by our test resource. Tests may leverage `LdapServerTestResource` provided by Quarkus as link:{quickstarts-blob-url}/security-ldap-quickstart/src/test/java/org/acme/elytron/security/ldap/ElytronLdapExtensionTestResources.java[we do] in the test coverage of the example application.
+<2> The URL used by our test resource. Tests may leverage `LdapServerTestResource` provided by Quarkus as link:{quickstarts-blob-url}/security-ldap-quickstart/src/test/java/org/acme/elytron/security/ldap/ElytronLdapExtensionTestResources.java[we do] in the test coverage of the example application.
+<3> `{0}` is substituted by the `uid`.
+<4> Without this configuration, every request to your service will cause an additional roundtrip to the LDAP server. Therefore, it's a common practice to cache these result to improve performance, but the tradeoff is that there will be a delay between changes in the LDAP getting effective in your service. The default cache max-age is `60s`.
```

Comment:
Thanks @t1, this information will be useful. 
However,  I'd like to propose to turn into a Note, that would inform users that by default,  LDAP request is made each time,  and you can set this property if you'd like to optimize, etc

The reason I propose it is because with this current  text we essentially recommend enabling the cache, in which case it would be reasonable for users to ask, why do we have to configure it ourselves?

The note, with pros and cons, would make it more obvious why users have to make this decision